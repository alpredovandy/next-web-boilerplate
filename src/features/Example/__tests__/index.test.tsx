/** TODO: Remove this file after use this example */
import React from "react";

import { render } from "@testing-library/react";

import ExampleFeature from "@features/Example";

describe("Test features: Example", () => {
  const renderExample = () => render(<ExampleFeature />);

  it("Snapshot", () => {
    const { container } = renderExample();

    expect(container).toMatchSnapshot();
  });
});
