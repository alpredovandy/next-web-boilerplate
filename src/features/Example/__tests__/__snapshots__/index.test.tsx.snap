// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test features: Example Snapshot 1`] = `<div />`;
